WorkflowSample/AppDelegate.swift

```diff
@@ -11,8 +11,6 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
<!--
  0 failure: 
  1 warning:  Consider using fi...
  
  
  DangerID: danger-id-Danger;
  File: WorkflowSample/AppDelegate.swift;
  Line: 12;
-->

- :warning: Consider using final for this class or use a struct (final_class)